### ISSUE TITLE
Invalid Change History - org.eclipse.jgit/src/org.eclipse.jgit.api.PullCommand#call()$if(232-235)

### ISSUE THREAD

**Issue by flozender** [jodavimehran/code-tracker#130]
In commit [`db55d13f5f4b641a9390076ced6b1775bcfcd6f6`](https://github.com/flozender/jgit/commit/db55d13f5f4b641a9390076ced6b1775bcfcd6f6?diff=split#diff-16ab513a4844e1f0235429b2e649147c9078966bd311558a00efeca8f9b82604R232) and file `org.eclipse.jgit/src/org/eclipse/jgit/api/PullCommand.java`, element `org.eclipse.jgit/src/org.eclipse.jgit.api.PullCommand#call()$if(232-235)` was identified as being `introduced`, which may not be accurate. 

Oracle filename: `jgit-PullCommand-call-IF_STATEMENT-9.json`

**Comment by flozender** [comment]
The mapping is with `L232` but should be with `L192`

**Comment by tsantalis** [comment]
@flozender 
There are two identical statements in the method.
```java
if (monitor.isCancelled())
			throw new CanceledException(MessageFormat.format(
					JGitText.get().operationCanceled,
					JGitText.get().pullTaskName));
```
At some point, these two should have been matched one with the other and vice-versa.
But I cannot find in which commit this happened.
I debugged commit-by-commit up to https://github.com/flozender/jgit/commit/770c733687d9f2f71f30822f9691427bf83b7577
but I didn't find any issue.
Can you help to find this problematic commit?

**Comment by flozender** [comment]
Yes, the problematic commit is [7668a46](https://github.com/flozender/jgit/commit/7668a462829b12e4f6a192c43157152ef63ff305#diff-16ab513a4844e1f0235429b2e649147c9078966bd311558a00efeca8f9b82604R198).

The mapping here is (`leftBlock <--> rightBlock`): 
```
if(232-235)<0544c9a> <--> if(198-201)<7668a46>
```

Hope this helps!

**Comment by tsantalis** [comment]
![Screenshot from 2023-10-18 00-14-56](https://github.com/jodavimehran/code-tracker/assets/1483516/61fa883a-6726-47d4-aae2-2cfda08dd9c1)

![Screenshot from 2023-10-18 00-16-48](https://github.com/jodavimehran/code-tracker/assets/1483516/8bc09654-bdc1-4a52-98a1-5016809afd45)
